playbooks/roles/jenkins/defaults/main.yml

```diff
@@ -109,10 +109,12 @@ jenkins_list_views:
       - notify-buyers-to-award-closed-briefs-8-weeks-production
       - notify-buyers-when-requirements-close-production
       - notify-suppliers-of-awarded-briefs-production
-      - notify-suppliers-of-dos-opportunities-production
+      - notify-suppliers-of-dos2-opportunities-production
+      - notify-suppliers-of-dos3-opportunities-production
       - notify-suppliers-of-new-questions-answers-production
       - notify-suppliers-of-brief-withdrawals-production
-      - upload-dos-opportunities-email-list-production
+      - upload-dos2-opportunities-email-list-production
+      - upload-dos3-opportunities-email-list-production
```

Comment:
I think we'll need to change the `build_monitor_jobs` listings as well - will they both fit?